<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_or_edit_procentovka</class>
 <widget class="QDialog" name="add_or_edit_procentovka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_addToDatabase">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Занести в базу данных</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <column>
         <property name="text">
          <string>Дата</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ФИО студента</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Введение</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Анализ треб-ний</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Конструкторский</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Документирование</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Эксперемент-ый</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Эконом-ий</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Заключение</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Работа ПП</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Презентация</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>id_stud</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonSformGrafProc">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сформировать график процентовки</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дата</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QSpinBox" name="spinBox_presentacia">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Анализ требований</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QSpinBox" name="spinBox_anTreb">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QSpinBox" name="spinBox_Document">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QSpinBox" name="spinBox_Zakl">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="spinBox_vved">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QSpinBox" name="spinBox_rabPP">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QSpinBox" name="spinBox_Eksperement">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введение</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDateEdit" name="dateEdit_data">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="displayFormat">
         <string>dd MMMM yyyy</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Конструкторский</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QSpinBox" name="spinBox_Konstr">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QSpinBox" name="spinBox_Ekonom">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Эксперементальный</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Экономический</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Заключение</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Работа ПП</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Презентация</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Документирование</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Студент</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_student">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
